app/view/VictoryPointMarker.svelte

```diff
@@ -1,39 +1,65 @@
 <script>
-import image from '../image/piece/vp';
-import { useScale } from '../context';
+  import { move } from 'ramda';
+  import context, { useScale } from '../context';
+  import { getVictoryPointImagePath } from '../util/image';
+  import propEq from 'ramda/src/propEq';
 
-const scale = useScale();
+  const scale = useScale();
+  const { state } = context();
 
-export let factions;
-export let x;
-export let y;
+  export let factions;
+  export let x;
+  export let y;
 
-$: boxShadow = new Array(factions.length)
-  .fill(0)
-  .map((_, i) => `0 ${8 * i + 3}px 0 rgb(179, 174, 166), 0 ${8 * i + 7}px 0 rgb(198, 185, 165), 0 ${8 * i + 8}px 0 rgb(113, 107, 97)`)
-  .join(',');
+  let factionsSorted;
+
+  $: {
+    let currentIndex = factions.findIndex(propEq('faction', $state.turn));
+    factionsSorted = currentIndex !== -1 ? move(currentIndex, factions.length - 1, factions) : factions;
+  }
 </script>
 
 <div
-  class='marker'
+  class="marker-stack"
   style={`
-    background-image: url(${image(factions[0])});
-    left: ${x}px;
-    top: ${y}px;
     transform: translate(-50%, -50%) translateY(-${8 * factions.length * $scale}px) scale(${$scale});
-    box-shadow: 0 0 0 2px rgba(0, 0, 0, 0.6) inset, ${boxShadow};
+    left: ${x}px;
+    top: ${y + factions.length * 10}px;
   `}
-  />
+>
+  {#each factionsSorted as faction, i }
+    <div
+      class="marker"
+      style={`
+        background-image: url(${getVictoryPointImagePath(faction.faction)});
+        top: -${158 * i}px;
+        transform: translateY(${150 * i}px);
+      `}
+    />
+  {/each}
+</div>
 
 <style>
-.marker {
-  position: absolute;
-  pointer-events: none;
-  user-select: none;
-  transform-origin: center;
-  width: 141px;
-  height: 141px;
-  border-radius: 20px;
-  transition: top 0.2s, left 0.2s;
-}
+  .marker-stack {
+    position: absolute;
+    width: 141px;
+    height: 141px;
+  }
+
+  .marker-stack:hover .marker {
+    transform: unset !important;
+  }
```

Comment:
I would prefer not to use `!important` for anything... I see no reason why it should be needed here

Reply:
We need it here, because transform that puts tokens in stack is set in style attribute

Reply:
Ah yes I see, thanks. This is fine then